src/components/SettingsDialog/Participants/Participants.scss

```diff
@@ -177,21 +185,26 @@
   color: $color-white;
   width: fit-content;
 
+  &.banned {
+    opacity: 0.2;
+    cursor: default;
+  }
```

Comment:
I think I would prefer if the roles are replaced by a "Banned" badge or something similar

Reply:
Is such a badge designed already?
Otherwise I'd argue that this should be a new issue and be designed at first, possibly with a change to the Participant/Moderator selection, too.
@SelinaBuff what's your opinion on it?